packages/bot-analytics/test/helpers/start-db.ts

```diff
@@ -0,0 +1,70 @@
+import waitPort from "wait-port";
+import { exec, spawn } from "child_process";
+import { PrismaClient } from "@prisma/client";
+
+const decoder = new TextDecoder("utf8");
+
+export const startDb = async () => {
+  const filePath = `${__dirname}/../../deploy/docker-compose.yml`;
+  const childProcess = spawn(`docker-compose`, ["--file", filePath, "up"]);
+
+  childProcess.stdout.on("data", (data) => {
+    console.log(decoder.decode(data));
+  });
+
+  childProcess.stderr.on("data", (data) => {
+    console.error(decoder.decode(data));
+  });
+
+  childProcess.on("close", (code) => {
+    console.log(`docker-compose exited with code=${code}`);
+  });
+
+  await waitPort({
+    port: 5432,
+  });
+
+  const url = `postgresql://postgres:postgres@localhost:5432/postgres?schema=mangrove`;
+  const prisma = new PrismaClient({
```

Comment:
Maybe having 2 dbs, one for tests and one for development

Reply:
Yes this is a good idea, will do